src/main/java/androidx/graphics/Bitmap.kt

```diff
@@ -111,3 +116,56 @@ inline fun createBitmap(
 ): Bitmap {
     return Bitmap.createBitmap(width, height, config, hasAlpha, colorSpace)
 }
+
+
+/**
+ * Returns ByteArray compressed from this bitmap with the specified [format]
+ * and [quality].
+ *
+ * @param format The format of bitmap.
+ * @param quality Hint to the compressor, 0-100. 0 meaning compress for small size, 100 meaning compress for max quality.
+ * @return ByteArray
+ */
+inline fun Bitmap.toByteArray(format:CompressFormat = CompressFormat.JPEG, @IntRange(from = 0, to = 100) quality: Int=100)
+        = ByteArrayOutputStream().also { compress(format, quality, it) }.toByteArray()
+
+/**
+ * Creates a new bitmap, clipped from this bitmap. If the specified [x],[y],
+ * [width],[height] are the same as the current width and height of this bitmap,
+ * this bitmap is returned and no new bitmap is created.
+ *
+ * @param x      The x coordinate of the first pixel.
+ * @param y      The y coordinate of the first pixel.
+ * @param width  The width.
+ * @param height The height.
+ * @return the clipped bitmap
+ */
+inline fun Bitmap.clip(x:Int, y:Int, width: Int, height: Int)
```

Comment:
You should run klint, your formatting is inconsistent (`x:Int` vs `with: Int` for instance)
I would rather call this method `crop()`